packages/features/ee/deployment/licensekey/CreateLicenseKeyForm.tsx

```diff
@@ -72,7 +72,7 @@ const CreateANewLicenseKeyFormChild = ({ session }: { session: Ensure<SessionCon
   function calculateMonthlyPrice() {
     const occurrence = watchedBillingPeriod === "MONTHLY" ? 1 : 12;
 
-    const sum = watchedEntityCount * watchedEntityPrice * occurrence;
```

Comment:
Is there a reason for this?

Reply:
Yeah this calc was wrong to how we calculate the totals - kept it in line with how it works for organizations too